pkgs/by-name/ta/task-master-ai/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+buildNpmPackage rec {
+  pname = "task-master-ai";
+  version = "0.15.0";
+
+  src = fetchFromGitHub {
+    owner = "eyaltoledano";
+    repo = "claude-task-master";
+    rev = "v${version}";
```

Since this repository uses a tag system, we may as well follow it:
```suggestion
    tag = "v${version}";
```